db/schema.rb

```diff
@@ -659,6 +659,15 @@
   add_index "organisation_visits", ["organisation_id"], name: "index_organisation_visits_on_organisation_id", using: :btree
   add_index "organisation_visits", ["visit_id", "organisation_id"], name: "index_organisation_visits_on_visit_id_and_organisation_id", unique: true, using: :btree
 
+  create_table "specifics", force: :cascade do |t|
```

Comment:
Happy to take em out for this PR, but we'll need em for reactions, and ideally we'd move webhooks to working this way as well. I kinda want to discourage schema changes in plugins if we can help it.